_aicspylibczi/Reader.cpp

```diff
@@ -307,10 +322,19 @@ namespace pylibczi {
   Reader::getMatches(SubblockSortable& match_)
   {
       SubblockIndexVec ans;
-      std::copy_if(m_orderMapping.begin(), m_orderMapping.end(), std::back_inserter(ans),
-          [&match_](const SubblockIndexVec::value_type& a_) {
-              return match_==a_.first;
-          });
+//      std::copy_if(m_orderMapping.begin(), m_orderMapping.end(), std::back_inserter(ans),
+//          [&match_](const SubblockIndexVec::value_type& a_) {
+//              return match_==a_.first;
+//          });
```

Comment:
delete the above commented out lines, as it looks like they are meant to be replaced by the code below it